lib/ios.js

```diff
@@ -230,8 +230,7 @@ class OptionalIfuseCommandCheck extends DoctorCheck {
   async diagnose () {
     const ifusePath = await resolveExecutablePath('ifuse');
     if (ifusePath) {
-      const {stdout, stderr} = await exec(ifusePath, ['-V']);
-      const version = `${stdout.trim()}${stderr.trim()}`;
+      const version = (await exec(ifusePath, ['-V'])).stderr;
       return okOptional(`ifuse is installed at: ${ifusePath}. Installed version is: ${/[0-9.]+/.exec(version)[0]}`);
```

Comment:
do we care if there's no match, so exec returns `null` ?

Reply:
good point.
I tried to install ifuse via github and brew. Both of them had the same format version number.
Thus, maybe the current way is pretty enough.
but yeah, considering `null` case is safer